pkg/controller/upload_controller.go

```diff
@@ -41,8 +41,13 @@ import (
 	"k8s.io/client-go/tools/leaderelection"
 	"k8s.io/client-go/tools/leaderelection/resourcelock"
 	"k8s.io/utils/clock"
+	"math"
+	"time"
 )
 
+const MAX_BACKOFF = 60
+const RETRY_WARNING_COUNT = 8
```

Comment:
Why do we need this one？

Reply:
MAX_BACKOFF used to limit 1 hour max backoff.
RETRY_WARNING_COUNT used to give a warning message for user to fix network issue if retry more than 8 times.